<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadingWidget</class>
 <widget class="QWidget" name="LoadingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoadingWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color: rgb(221, 221, 221);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="info_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Info label</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="info_label_icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="load_animation_label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
